profiles/killbill/src/test/java/org/killbill/billing/server/log/obfuscators/TestPatternObfuscator.java

```diff
@@ -17,16 +17,112 @@
 
 package org.killbill.billing.server.log.obfuscators;
 
-import org.killbill.billing.server.log.ServerTestSuiteNoDB;
+import java.util.Collection;
+import java.util.List;
+
 import org.mockito.Mockito;
 import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 import ch.qos.logback.classic.spi.ILoggingEvent;
 
-public class TestPatternObfuscator extends ServerTestSuiteNoDB {
+public class TestPatternObfuscator {
+
+    private PatternObfuscator obfuscator;
+
+    @BeforeClass(groups = "fast")
+    public void beforeClass() throws Exception {
+        System.setProperty("killbill.server.log.obfuscate.keywords", "req.requestURI,accountnumber,authenticationdata,banknumber,bic,cardvalidationnum,cavv,ccFirstName,ccLastName,ccNumber,ccTrackData,ccVerificationValue,ccvv,cvNumber,cvc,cvv,email,iban,name,number,password,xid,cell_phone");
+        System.setProperty("killbill.server.log.obfuscate.patterns.separator", "#");
+        System.setProperty("killbill.server.log.obfuscate.patterns", "\\s*=\\s*'([^']+)'#\\s*=\\s*\\\"([^\\\"]+)#\\s*=\\s*([^ '\\\",]+)#\\s*:\\s*'([^'\\\",{}]+)'");
+        obfuscator = new PatternObfuscator();
+    }
+
+    @Test
```

Comment:
Can you add the `groups = "fast"` for all tests?

Reply:
Done